components/dropdown/dropdown-content-mixin.js

```diff
@@ -244,7 +275,18 @@ export const DropdownContentMixin = superclass => class extends RtlMixin(supercl
 	}
 
 	close() {
-		this.opened = false;
+		const hide = () => {
+			this._closing = false;
+			this.shadowRoot.querySelector('.d2l-dropdown-content-width').removeEventListener('animationend', hide);
```

Comment:
According to [caniuse](https://caniuse.com/?search=once), it should be possible to avoid the explicit removal if the `once` parameter is passed when adding.

```javascript
this.shadowRoot.querySelector('.d2l-dropdown-content-width')
    .addEventListener('animationend', hide, { once: true });
```

However, Mozilla seems to indicate that was added in Edge 18, and it might be hard to find Edge 17.

Reply:
Looks like the once parameter is being used in other spots in the lms like https://search.d2l.dev/xref/BrightspaceUI/htmleditor/htmleditor.js?r=fd11e9a4#297, so it seems safe to use. I can make that change.